### PR TITLE
GQL-98: Implements AWS CDK and removes Serverless Framework

### PR DESCRIPTION
**Pull request by macrouch**
# Overview

### What is the feature?

Implements AWS CDK and removes Serverless Framework
Changes the local dev environment to use AWS SAM Local (see README updates)
Updates environment variable config to simplify local environment setup

### What areas of the application does this impact?

Local dev environment, deployments

# Testing

Follow updates in README to install Prerequisites and start local environment

# Checklist

- [x] I have added automated tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,33 +21,53 @@ CMR-GraphQL is an API developed by [NASA](http://nasa.gov) [EOSDIS](https://eart
 
 ## Application Installation
 
-Before running the application you'll want to ensure that all necessary packages are installed by running:
+### Prerequisites
 
-   npm install
+- Docker
+- aws-sam-cli (`brew install aws-sam-cli`)
```
Comment (mandyparson) [inline README.md:27]: I also needed into install dotenvx (brew install dotenvx/brew/dotenvx) and aws-cdk (brew install aws-cdk)

Comment (macrouch) [reply README.md:27]: Those should come from `npm install`, I don't have those installed with brew in my machine. You might need to wait for the @edsc/cdk-utils library to be published

**Review by dmistry1 — APPROVED** [review]

File: cdk/graphql/lib/graphql-stack.ts [hunk 2]
```diff
@@ -0,0 +1,207 @@
...
+
+    /**
+     * EarthdataVarinfo Lambda resources
+     */
+    const earthdataVarinfoLogGroup = new logs.LogGroup(this, 'EarthdataVarinfoLogGroup', {
+      logGroupName: `/aws/lambda/${this.stackName}-earthdataVarinfo${logGroupSuffix}`,
+    });
+
+    const earthdataVarinfoLambdaFunction = new lambda.Function(this, 'EarthdataVarinfoLambdaFunction', {
+      // TODO does this work in dev?
```
Comment (eudoroolivares2016) [inline cdk/graphql/lib/graphql-stack.ts:132]: I was trying this myself in dev I was not able to get it to run these

**Review by eudoroolivares2016 — APPROVED** [review]
I was able to get requests working I can swtich envs using the different npm starts
The only parts that I was not able to get working are using edl tokens was causing me issues though I'm pulling in the .env file so I was not able to test the generate variable drafts

**Pull request merged**